packages/php-wasm/web-service-worker/src/utils.ts

```diff
@@ -229,3 +296,9 @@ export function getRequestHeaders(request: Request) {
 	});
 	return headers;
 }
+
+export function removeContentSecurityPolicyDirective(directive: string) {
```

Comment:
Is this used anywhere? Or should we remove it?

Reply:
Ah, right. It represented intent to test. I just filled this in and made it an export so we can test it.